wikichange/src/popup/popup.js

```diff
@@ -0,0 +1,6 @@
+document.addEventListener('DOMContentLoaded', function () {
+    let compsLink = document.getElementById("comps-website");
```

Comment:
nit: can we do `const` instead of `let`? better to not have anything be `let` if it's not going to be mutated.